lua/no-neck-pain/config.lua

```diff
@@ -148,6 +152,13 @@ function NoNeckPain.setup(options)
         NoNeckPain.options.buffers.right.backgroundColor
     ) or NoNeckPain.options.buffers.backgroundColor
 
+    NoNeckPain.options.buffers.textColor = NoNeckPain.options.buffers.textColor
+        or NoNeckPain.options.buffers.backgroundColor
+    NoNeckPain.options.buffers.left.textColor = NoNeckPain.options.buffers.left.textColor
+        or NoNeckPain.options.buffers.textColor
+    NoNeckPain.options.buffers.right.textColor = NoNeckPain.options.buffers.right.textColor
+        or NoNeckPain.options.buffers.textColor
```

Comment:
```suggestion
    NoNeckPain.options.buffers.textColor = NoNeckPain.options.buffers.textColor
        or NoNeckPain.options.buffers.backgroundColor
    NoNeckPain.options.buffers.left.textColor = NoNeckPain.options.buffers.left.textColor
        or NoNeckPain.options.buffers.left.backgroundColor
    NoNeckPain.options.buffers.right.textColor = NoNeckPain.options.buffers.right.textColor
        or NoNeckPain.options.buffers.right.backgroundColor
```

correct?

And could you update the tests in `test_API.lua`? the ones that cover this are `sets exposed methods and default options value` and `overrides default values`

Reply:
I think the priority order should be:
- buffer specific text color
- general text color
- background color

What do you think?

Reply:
Definitely! Great catch